charts/agent/values.yaml

```diff
@@ -1,10 +1,18 @@
-# -- The number of replicas for the deployment
-replicaCount: 2
```

Comment:
Moving `replicaCount` under `deployment` is a breaking change for users which have altered this value.

In most charts `replicaCount` is defined at the root - maybe keep it like this and avoid the breaking change?

Reply:
@pat-s In my mind, updating a number of replica of a deployment it's not really a breaking change.
I just wanted to stay consistent with the similar part on the server component (`.Values.statefulSet.replicaCount`).
But it's also possible to change the two chart to put these values at the root of every component.
Let me know what the woodpecker team prefer.

Reply:
It's just a thought of mine, I am not making the decisions here :)

> @pat-s In my mind, updating a number of replica of a deployment it's not really a breaking change.

From a mapping PoV it is, i.e. previous explicit declarations won't work as before after this change. Hence it's "breaking" in that sense. It wouldn't breaking if it were an addition or a change of a default value. I am not saying that it would be a problem - most likely nobody has explicitly defined this value. 

Let's see what @anbraten has to say.